android-incremental-ota.py

```diff
@@ -0,0 +1,28 @@
+import os
+print("Incrementals are made using target zips which are found under out/target/product/device/obj/PACKAGING/target_files_intermediates") 
```

Comment:
restyle the whole program with spaces at appropriate places. Refer this link https://www.python.org/dev/peps/pep-0008/